test/controllers/api/professors/introductions/create_test.rb

```diff
@@ -0,0 +1,77 @@
+require 'test_helper'
+
+class Api::Professors::IntroductionsControllerCreateTest < ActionDispatch::IntegrationTest
+  setup do
+    @lo = FactoryBot.create(:lo)
+  end
+
+  context 'create' do
+    context 'with valid params' do
+      should 'be successfully' do
+        introduction_attributes = FactoryBot.attributes_for(:introduction, lo: @lo)
+        post api_professors_lo_introductions_path(@lo), params: { introduction: introduction_attributes }, as: :json
```

Comment:
Need to be refactor all the tests, you don't pass the position and the lo_id. Also need to refactor the messages